### DISCUSSION TITLE
MACs with Multiple IPs

### DISCUSSION THREAD

**Discussion by photomatix18** [netalertx/NetAlertX#264] [Q&A]
Hello,

How are MACs handled with multiple IPs? I'm running Unraid and I'm getting only getting one device for the main NIC listed in PI.Alert. I'm assuming it's the first one that responds via ARP and it's not Unraid's IP.

**Comment by jokob-sk** [comment]
Hey,

The last IP detected via arp-scan is stored against the MAC.

j

**Comment by photomatix18** [comment]
Gotcha! Are there any options for adding multiple entries for 1 MAC? Or is that something that may be coming in a future release? I'm no expert, so I'm not really sure how to explain why this happens, but for docker containers that have a dedicated IP, they share the same MAC address as the host in Unraid.

**Comment by jokob-sk** [comment]
Hey, 

Currently not planned or on the roadmap, but it could possibly be achieved via a custom plugin if someone would like to submit a PR :)

j
